docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,100 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+## Database Backend
 
-_Here we provided a small sample on K8s_
+You can install [Crunchy PostgreSQL from OperatorHub.io][crunchy].  After the
+installation, to create a PostgreSQL cluster, run this command:
 
-## OpenShift
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/pgcluster.yaml
```

Comment:
I fear there is a bit too much "magic" behind this single yaml to apply.
I think it's important to explain how one would do it, but for the sake of the simplicity, we are providing a ready to use template. 
We should really explain to the user what he is doing.

Reply:
Done. 22659064cbf6055a24cbb747dd9e4f5e978037f8